TaskfMRIAnalysis/scripts/makeSubjectTaskSummary.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+#~ND~FORMAT~MARKDOWN~
+#~ND~START~
+#
+# # makeSubjectTaskSummary.sh
+#
+# ## Copyright (C) 2021 The Human Connectome Project
+#
+# * Washington University in St. Louis
+#
+# ## Author(s)
+#
+# * Greg Burgess, Washington University in St. Louis
+#
+# ## Product
+#
+# [Human Connectome Project][HCP] (HCP) Pipelines
+#
+# ## License
+#
+# See the [LICENSE](https://github.com/Washington-University/Pipelines/blob/master/LICENCE.md) file
+#
+# ## Description
+#
+# This script will create subject-level summary directory containing images required for
+# group-level analyses. Script reads arguments set in TaskfMRIAnalysis.sh to determine
+# which analyses were run (grayordinates, parcellated, volume) and at which "level"
+# (Level1 vs. Level2). Script creates symlinks to Level2 outputs (if they exist) or
+# outputs from single Level1. Unlike film / flameo, this script will use identical naming
+# convention for Level1 and Level2 outputs. This allows group-level analyses to include
+# subjects whether their outputs included one run or multiple runs.
+#
+#
+# <!-- References -->
+# [HCP]: http://www.humanconnectome.org
+# [FSL]: http://fsl.fmrib.ox.ac.uk
+#
+#~ND~END~
+
+set -eu
+
+# ------------------------------------------------------------------------------
+#  Check that HCPPIPEDIR is defined and Load Function Libraries
+# ------------------------------------------------------------------------------
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/opts.shlib"			# Function for getting HCPpipelines version
```

Comment:
Do not source `opts.shlib` if you are sourcing/using `newopts.shlib`, `opts.shlib` should be considered legacy-only code at this point, and they should not be mixed.  As for the comment that appears to explain why `opts.shlib` is being sourced, run "$HCPPIPEDIR/show_version" if you need to get the pipelines version info.

Reply:
This is addressed in next commit.